src/application.rs

```diff
@@ -74,11 +81,5 @@ pub fn create_app() -> Result<rocket::Rocket, Box<dyn Error>> {
         routes::syntaxes::get_syntaxes,
         routes::snippets::import_snippet,
     ];
-    Ok(app
-        .manage(config)
-        .manage(storage)
-        .manage(token_validator)
-        .attach(RequestIdHeader)
-        .attach(RequestSpan)
```

Comment:
Where did `RequestSpan` go?

Reply:
That's the annoying part: we can no longer use our implementation that was based on thread-locals, as Tokio can move futures around different threads while a request is being executed (e.g. in `await` for a DB request) :( I don't see a way to implement this properly on our end: my understanding is that Rocket needs to create a top-level tracing span. There is a feature request for that, but the work has been delayed because of the 0.5 migration. They expect it to land in 0.6.